src/core/qgsvector3d.h

```diff
@@ -18,6 +18,7 @@
 
 #include "qgis_core.h"
 #include "qgis.h"
+#include "qgspoint.h"
```

Comment:
why is this include required to the header if there is no change.
can it be move where appropriate?

Reply:
Oops, an unsuccessful trial residue, I delete.